CSharp/DevVmPowershell/Helpers/ImportApiHelper.cs

```diff
@@ -0,0 +1,215 @@
+		{
+			DataTable dataSource = new DataTable();
+
+			switch (fileType.ToLower())
+			{
+				case Constants.FileTypes.Documents:
+					dataSource.Columns.Add(Constants.CommonFields.ControlNumber, typeof(string));
+					dataSource.Columns.Add(Constants.CommonFields.FilePath, typeof(string));
+					dataSource.Columns.Add(Constants.CommonFields.ParentDocId, typeof(string));
+					dataSource.Columns.Add(Constants.CommonFields.FileName, typeof(string));
+					break;
+				case Constants.FileTypes.Images:
+					dataSource.Columns.Add(Constants.CommonFields.Bates, typeof(string));
+					dataSource.Columns.Add(Constants.CommonFields.Doc, typeof(string));
+					dataSource.Columns.Add(Constants.CommonFields.File, typeof(string));
+					break;
+			}
+
+			AddFilesToDataTable(dataSource, fileType, fileCount, currentFileCount);
+
+			return dataSource;
+		}
+
+		public static void AddFilesToDataTable(DataTable dataSource, string fileType, int fileCount, int currentFileCount)
+		{
+			string executableLocation = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+			string resourcePath = Path.Combine(executableLocation, $@"Resources\{fileType}");
+			string[] files = Directory.GetFiles(resourcePath);
+
+			for (int i = 0; i < fileCount;)
+			{
+				int startingIndex = i;
+				foreach (string filePath in files)
+				{
+					switch (fileType.ToLower())
+					{
+						case Constants.FileTypes.Documents:
+							dataSource.Rows.Add($"DOC_{currentFileCount + i}", filePath, "", Path.GetFileName(filePath));
+							break;
+						case Constants.FileTypes.Images:
+							dataSource.Rows.Add($"IMG_{currentFileCount + i}", $"IMG_{currentFileCount + i}", filePath);
+							break;
+					}
+					i++;
+					if (i >= fileCount)
+					{
+						break;
+					}
+				}
+
+				if (i == startingIndex)
+				{
+					break;
+				}
+			}
+		}
+
+		static void ImportJobOnMessage(Status status)
+		{
+			Console.WriteLine($"Message: {status.Message}");
+		}
+
+		static void ImportJobOnFatalException(JobReport jobReport)
+		{
+			Console.WriteLine($"Fatal Error: {jobReport.FatalException}");
+		}
+
+		static void ImportJobOnComplete(JobReport jobReport)
+		{
+			Console.WriteLine($"Job Finished With {jobReport.ErrorRowCount} Errors: ");
+		}
+
+		static async Task<int> GetNumberOfDocumentsAsync(ServiceFactory serviceFactory, int workspaceId, string fileType)
+		{
+			using (IObjectManager objectManager = serviceFactory.CreateProxy<IObjectManager>())
+			{
+				QueryRequest queryRequest = new QueryRequest();
+				queryRequest.ObjectType = new ObjectTypeRef() { Name = "Document" };
+				queryRequest.Fields = new List<FieldRef>()
+				{
+					new FieldRef() {
+						Name = "Control Number"
+					},
+					new FieldRef() {
+						Name = "Has Images"
+					},
+					new FieldRef() {
+						Name = "Has Native"
+					}
+				};
+
+				switch (fileType.ToLower())
+				{
+					case Constants.FileTypes.Documents:
+						queryRequest.Condition = "(('Control Number' LIKE 'DOC_'))";

```

Comment:
Can we instead use CommonFields from Constants class for field names to be consistent?